.changes/unreleased/tinymce-TINY-10885-2024-04-30.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: Insert/Edit image dialog lost focus after the image upload completed.
```

I think we would usually use the `would` word in here so something like:
```suggestion
body: Insert/Edit image dialog would lose focus after the image upload completes.
```

---

The only real requirement is that it's past tense - so what I wrote and your suggestion are both valid. It comes down to personal choice.

---

@TheSpyder, could you please revise this improvement as a native speaker?

---

@vpyshnenko I prefer my suggestion, that's why I wrote it :)

If we think it needs rewording, how about
```suggestion
body: The Insert/Edit image dialog no longer loses focus after image uploads complete.
```

---

I'll stick your original one since it is in the past tense